frame.scad

```diff
@@ -9,10 +9,12 @@ use <belt.scad>;
 
 thickness = 4;
 width = 30;
-height_at_front = 120;
-height_at_back = 140;
-cube_to_arm_gap = 60;
-cube_to_arm_len = 80;
+
+// TODO import this?
+claw_clips_length = claw_wall_thickness*7;
```

Comment:
This variable considers the length of the snaps and then the thickness above?  Might be a clearer way to put this...